ember-accounts/app/templates/components/update-email.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="fullscreen-credit-card">
+  <form {{action 'update' on="submit"}}>
+    <div class="modal-header">
+      <button type="button" class="close" {{action 'close'}} data-dismiss="modal" aria-hidden="true">×</button>
+      Update Account Email
+    </div>
+    <div class="modal-body">
+      <fieldset>
+        <div class="form-row full">
+          <label>
+            <span>Email Address</span>
+          </label>
+          {{input value=email type="email" required=true}}
```

Comment:
Often fields like this are pre-filled with the current value. Is that possible here?